src/configs/encoders.py

```diff
@@ -0,0 +1,198 @@
+# Copyright 2021 Google LLC
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#     https://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Transformer Encoders.
+Includes configurations and factory methods.
+"""
+from typing import Optional
+
+import dataclasses
+import gin
+import tensorflow as tf
+
+from official.modeling import hyperparams
+from official.modeling import tf_utils
+from official.nlp.modeling import layers
+from official.nlp.modeling import networks
```

Comment:
Nit: these imports can be grouped with the block above (but inserted alphabetically).